src/requirements/EthereumAttestation/components/EthereumAttestation.tsx

```diff
@@ -0,0 +1,119 @@
+import { FormControl, FormLabel, Input } from "@chakra-ui/react"
+import FormErrorMessage from "components/common/FormErrorMessage"
+import { Chain } from "connectors"
+import { useFormContext, useWatch } from "react-hook-form"
+import { RequirementFormProps } from "requirements"
+import ChainPicker from "requirements/common/ChainPicker"
+import parseFromObject from "utils/parseFromObject"
+import { EAS_SCAN_BASE } from "../EthereumAttestationRequirement"
+
+const HEX_STRING_REGEX = /^0x[A-F0-9]+$/i
+const ADDRESS_REGEX = /^0x[A-F0-9]{40}$/i
```

Comment:
nit: we export this regex from another file, we can use it from there, so we have a "single source of truth" (I know it's located in a guild checkout related file, maybe we'll move it to a more appropriate file in the future)

Reply:
Thanks! Definitely better to have this defined only once! Made the change in https://github.com/agoraxyz/guild.xyz/pull/873/commits/0c49d0a10ae6d877b4a03a8e00dacb6b5e4a23c5 !